diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -10,7 +10,7 @@ import org.jetbrains.kotlin.com.intellij.lang.ASTNode
  * This class represent individual inspections of diktat code style.
  * A [Warnings] entry contains rule name, warning message and is used in code check.
  */
-@Suppress("ForbiddenComment", "MagicNumber", "TOO_MANY_PARAMETERS")
+@Suppress("ForbiddenComment", "MagicNumber")
```

Comment:
Why? We already have similar suppresses for detekt's inspection here, and need one for ours too